src/main/java/org/broadinstitute/consent/http/db/LibraryCardDAO.java

```diff
@@ -53,7 +53,13 @@ void updateLibraryCardById(@Bind("libraryCardId") Integer libraryCardId,
 
   @RegisterBeanMapper(value = LibraryCard.class)
   @UseRowReducer(LibraryCardReducer.class)
-  @SqlQuery("SELECT * FROM library_card WHERE id = :libraryCardId")
+  @SqlQuery("""
+      SELECT lc.*,
+      ld.daa_id
+      FROM library_card AS lc
+      LEFT JOIN lc_daa ld ON lc.id = ld.lc_id
+      WHERE id = :libraryCardId
```

Comment:
Should we have the alias here to be clear?
```suggestion
      WHERE lc.id = :libraryCardId
```

Reply:
Also, should this be a separate method? I imagine there are cases where we want the daa and the library cards, and other times where we just want the library cards.

Reply:
The user stories we are trying to support here are:
1. Researcher sees their LCs with DAA associations on their profile page
2. Signing Officials see a list of users along with which DAAs they are associated to (see table mocks) so the SO can select/unselect a user's DAA association
3. Researcher sees their DAA associations related to the datasets they have selected on a data access request (which comes from their LC). 
4. Researcher can request a DAA assignment from their SO via the profile page.

In all of these cases, we do need the DAA ids in some way. We don't always need the fleshed out DAA entity on the library card, but sometimes we will, so I think we do need the above method as it is, and we probably need another that fleshes out the actual DAAs on the library card.

Reply:
I've created a [ticket](https://broadworkbench.atlassian.net/browse/DUOS-3030) for a separate method when we need all details of the DAAs on the library card.